charts/osm/README.md

```diff
@@ -88,7 +88,6 @@ The following table lists the configurable parameters of the osm chart and their
 | osm.featureFlags.enableEgressPolicy | bool | `true` | Enable OSM's Egress policy API. When enabled, fine grained control over Egress (external) traffic is enforced |
 | osm.featureFlags.enableEnvoyActiveHealthChecks | bool | `false` | Enable Envoy active health checks |
 | osm.featureFlags.enableIngressBackendPolicy | bool | `true` | Enables OSM's IngressBackend policy API. When enabled, OSM will use the IngressBackend API allow ingress traffic to mesh backends |
-| osm.featureFlags.enableMulticlusterMode | bool | `false` | Enable Multicluster mode. When enabled, multicluster mode will be enabled in OSM |
```

Comment:
One thought: does removing these options violate our semver support guarantees? I believe all the multi-cluster stuff was alpha, but I do think this does break installation for users

Reply:
I'll let others comment on
@trstringer @shashankram

Reply:
@keithmattix how does this break installation?
We can always leave this feature flag behind as a no-op to be extra cautious.

Reply:
I think we could just add this to the release notes as well

Reply:
Sorry I was thinking more generally about removing options/feature flags from meshconfig

Reply:
ya he's right, removing from a CRD warrants a version change. Leaving the field in for now, and marking as deprecated

Reply:
Even if the field is kept in the CRD, removing the value here will break users of the chart if they either have existing scripts setting it or they upgrade with `helm upgrade --reuse-values`, either of which they would likely expect to succeed if the chart's major version is the same.

I think it's likely though that the number of people we'd break is literally zero with this change, but it's something to consider whenever making a change like this.

Reply:
Agreed! Maybe we can say that support between versions is non-breaking when using the CLI directly, but if they play around with the underlying helm chart we don't offer guaranteed compatibility. That way we can add additional logic to the CLI.

I also think it could make sense to provide additional steps for upgrades. That is, an upgrade is backwards compatible, but you may have to follow some small set of steps. We should probably also say you can only upgrade from 1 version to +2 minor versions at a time, or something along those lines.